pkg/crd/client.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2018 The original author or authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package crd
+
+import (
+	"fmt"
+	"github.com/projectriff/riff/pkg/env"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type riffClient struct {
+	restClient rest.Interface
+}
+type Client interface {
+	Create(obj *Manifest) (*Manifest, error)
+	Update(obj *Manifest) (*Manifest, error)
+	Delete(obj *Manifest) (*Manifest, error)
+	Get() (*Manifest, error)
+}
```

Comment:
missing `List()`

If we generate the client, we don't have to implement list by hand.